SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -50,6 +50,51 @@ DECLARE_CYCLE_STAT(TEXT("Sender UpdateInterestComponent"), STAT_SpatialSenderUpd
 DECLARE_CYCLE_STAT(TEXT("Sender FlushRetryRPCs"), STAT_SpatialSenderFlushRetryRPCs, STATGROUP_SpatialNet);
 DECLARE_CYCLE_STAT(TEXT("Sender SendRPC"), STAT_SpatialSenderSendRPC, STATGROUP_SpatialNet);
 
+namespace
+{
+struct FChangeListPropertyIterator
```

Comment:
Thoughts on also using this iterator in ComponentFactory? @mironec

Reply:
Or something similar - which also has the parent (I think that's all that's needed?)

Reply:
I didn't want to invade the component factory with event tracing, especially since it's also changing - what do you think?

Reply:
I like it! Not sure if I will get to it, though.

Reply:
That's ok - I can pull it over some time in the future! Will leave it for now whilst it's being refactored though